source/Dig/base.gfx.bitmapfont.bmx

```diff
@@ -2440,7 +2499,7 @@ End Type
 
 ' - max2d/max2d.bmx -> loadimagefont
 ' - max2d/imagefont.bmx TImageFont.Load ->
-Function LoadTrueTypeFont:TImageFont( url:Object,size:Int,style:Int )
+Function LoadTrueTypeFont:TImageFont( url:Object,size:Float,style:Int )
 	Local src:TFont = TFreeTypeFont.Load( String( url ), size, style )
```

Comment:
Ich bekomme hier eine Compiler-Warnung, dass der Parameter float ist, die Signatur aber int verlangt

Reply:
Ja...du musst brl.mod und pub.mod aktualisieren...damit gehen dann Kommazahlenschriftgroessen.

Reply:
(Nach dem Update der beiden Pakete auf den master kamen in denen ähnliche Warnungen)